freshmaker/handlers/botas/botas_shipped_advisory.py

```diff
@@ -237,9 +240,25 @@ def _handle_auto_rebuild(self, db_event):
             if not bundles:
                 self.log_warn('The bundle digest %r was not found in Pyxis. Skipping.', digest)
                 continue
+            bundle_nvr = bundles[0]['brew']['build']
+
+            # If specific container images where requested to rebuild, process only them
+            if (isinstance(self.event, ManualBundleRebuild)
+                    and bundle_nvr not in self.event.container_images):     # noqa: W503
```

Comment:
You need to check if `container_images` is empty or not, otherwise all bundles will be ignored in case of `container_images` is not specified.

Reply:
fixed